CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Changed
 - setmetatable no longer requires a second argument.
 - `allow_unused_self` is now toggled on for `unused_variable` by default.
+- unbalanced assignments with ellipsis no longer lints

```

Comment:
Last nitpick before merge--this was a bug, so it should be under `Fixed`, as well as with a period at the end.

Reply:
No problem! Done.